cmd/minikube/cmd/start.go

```diff
@@ -405,16 +406,25 @@ func virtualBoxMacOS13PlusWarning(driverName string) {
 	if !driver.IsVirtualBox(driverName) || !detect.MacOS13Plus() {
 		return
 	}
-	suggestedDriver := driver.HyperKit
-	if runtime.GOARCH == "arm64" {
-		suggestedDriver = driver.QEMU
-	}
-	out.WarningT(`Due to changes in macOS 13+ minikube doesn't currently support VirtualBox. You can use alternative drivers such as docker or {{.driver}}.
+	out.WarningT(`Due to changes in macOS 13+ minikube doesn't currently support VirtualBox. You can use alternative drivers such as 'vfkit', 'qemu', or 'docker'.
```

Comment:
is there a reason to do the virtuablox related changes in this PR ?

Reply:
Yes, otherwise virtaulbox message recommends hyperkit an replacement.